v2/pkg/commands/build/base.go

```diff
@@ -165,8 +165,22 @@ func (b *BaseBuilder) CompileProject(options *Options) error {
 		}
 	}
 
+	commands := slicer.String()
+
+	compiler := options.Compiler
+	if options.Obfuscate {
+		if !shell.CommandExists("garble") {
+			options.Logger.Print("garble not found. Please install it with `go install mvdan.cc/garble@latest`. Using Go compiler instead.")
```

Comment:
Should be promote this to an error?
Let's assume someone has ci/cd integration for building their project and they use the obfuscate option in `wails.json`, but forgot to install garble in their build agent environment. In such a case I would rather expect the build to fail instead of having a successful build which has no obfuscation active, and might be released undetected.

Reply:
💯 Really great point 👍